### PR TITLE
Merge 3.9 to 4.0

### PR DESCRIPTION
**Pull request by keshavprasadms**
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: ansible/roles/graylog-mongodb-import/defaults/main.yml [hunk 1]
```diff
@@ -1,29 +1,4 @@
...
-  - pipeline_processor_rules
-  - roles
-  - scheduler_job_definitions
-  - scheduler_triggers
-  - searches
-  - streamrules
-  - streams
-  - users
-  - views
+graylog_mongo_collections: "cluster_config, content_packs, content_packs_installations, event_definitions, event_notifications, event_notification_status, grants, grok_patterns, index_sets, inputs, lut_caches, lut_data_adapters, lut_tables, pipeline_processor_pipelines, pipeline_processor_pipelines_streams, pipeline_processor_rules, roles, scheduler_job_definitions, scheduler_triggers, searches, streamrules, streams, users, views"
```
Comment (rjshrjndrn) [inline ansible/roles/graylog-mongodb-import/defaults/main.yml:4]: Please keep it as an ansible list. It'll be more readable, and idiomatic.
Else it'll become like our yarn provision ansible role.

File: ansible/roles/graylog-mongodb-import/tasks/main.yml [hunk 2]
```diff
@@ -9,15 +9,8 @@
   with_items:
   - "{{ graylog_mongo_collections.split(',') }}"
```
Comment (rjshrjndrn) [inline ansible/roles/graylog-mongodb-import/tasks/main.yml:10]: We don't have to do a split, if it's a list

**Review by rjshrjndrn — CHANGES_REQUESTED** [review]

File: ansible/roles/graylog-mongodb-import/tasks/main.yml (hunk 2 above)
Comment (rjshrjndrn) [reply ansible/roles/graylog-mongodb-import/tasks/main.yml:10]: will refactor it in future release.

**Review by rjshrjndrn — APPROVED** [review]

File: ansible/roles/graylog-mongodb-import/defaults/main.yml (hunk 1 above)
Comment (keshavprasadms) [reply ansible/roles/graylog-mongodb-import/defaults/main.yml:4]: Sure we will change in next iteration as it needs changes acorss multiple files.

**Pull request merged**